api/app/views/mno_enterprise/jpi/v1/admin/organizations/_organization.json.jbuilder

```diff
@@ -1,2 +1,2 @@
 json.extract! organization, :id, :name, :uid, :soa_enabled, :created_at, :account_frozen, :financial_metrics, :billing_currency, :external_id, :belong_to_sub_tenant, :belong_to_account_manager, :demo_account
-
+json.role @user.role(organization) if @user
```

Comment:
Since `@user` is being passed into the partial in the view (...admin/organizations/show.json.jbuilder), can it be referred to as just `user` here or can the `user: @user` be removed from the show view?